pkg/sinks/sink.go

```diff
@@ -105,6 +104,33 @@ func (u *SinkProcessor) Start(ctx context.Context) error {
 			}
 			readers = append(readers, reader)
 		}
+
+		if u.VertexInstance.Vertex.Spec.Watermark.Disabled {
+			names := u.VertexInstance.Vertex.GetToBuffers()
+			if u.VertexInstance.Vertex.IsASink() {
+				// Sink has no to buffers, we use the vertex name as the buffer writer name.
+				names = append(names, u.VertexInstance.Vertex.Spec.Name)
```

Comment:
Already know it's a sink, then the line above is not needed?

Reply:
done